lib/identity/client.js

```diff
@@ -1278,6 +1280,29 @@ class Client extends PartialClient {
     await checkStatus(response)
     return AccessRequestApprovalsResponse.decode(await response.json())
   }
+
+  /**
+   * availableAccessRequestGroups returns Realm Groups with MPC Enabled
+   *
+   * @param {string} realmName - name of realm
+   *
+   * @return {Object} The Groups
+   */
+  async availableAccessRequestGroups(realmName) {
+    const request = new AccessRequestGroupsRequest(realmName)
+    let response = await this.storage.authenticator.tsv1Fetch(
+      `${this.storage.config.apiUrl}/v1/identity/pam/groups`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: request.stringify(),
+      }
+    )
+    await checkStatus(response)
+    return AccessRequestGroupsResponse.decode(await response.json())
```

Comment:
i'd say same goes for the response in this case. i'd just return an array of `{id, groupName}` group info